src/main/java/com/example/doodle_clone/models/Meeting.java

```diff
@@ -1,169 +1,39 @@
 package com.example.doodle_clone.models;
 
 import jakarta.persistence.*;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 import java.util.List;
 
 @Entity
-
+@Getter @Setter
+@NoArgsConstructor
```

Comment:
You should also add `@AllArgsConstructor` here and remove manual constructor

Reply:
Same here, comment not fixed

Reply:
> Same here, comment not fixed

This not a AllArgConstr(One variable is not in the constructor)  this can be a RequiredArgsConstructor,but this variable cant be `final`.I will now try to find how to create a constructor with the necessary variables in Lombook

Reply:
Got it then